spec/features/app_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative '../spec_helper'
+
+RSpec.describe 'App' do
+  def app
+    App
+  end
+
+  it "loads the home page" do
+    get '/'
+    expect(last_response).to be_ok
+  end
+
+  it "loads error page" do
+    get '/error'
+    expect(last_response).to include{"<h1>Error, cannot divide by zero!</h1>"}
+  end
+
+  it "checks if the form loaded" do
+    get '/'
+    expect(last_response.body).to include{"<form class='calculator-container' action='/calculate' method='post'>"}
+  end
+
+  it "checks if the input fields loaded" do
+    get '/'
+    expect(last_response.body).to include{"<input required='true' type='number' step='0.01' placeholder='Number 1' name='input_one' value='<%= @result %>'/>"}
+    expect(last_response.body).to include{"<input required='true' type='number' step='0.01' placeholder='Number 2' name='input_two'/>"}
+  end
+
+  it "checks if the operator buttons loaded" do
```

Comment:
I would change all the messages in the spec for something more imperative. Like, instead of `checks if something happen` I would write `something happens`.

```suggestion
  it "renders operation buttons" do
```

Reply:
Wow, reading up on capybara now and it fits perfectly for what I was testing.  The Unit tests in this specific case felt really redundant, I'll try and implement this and better the tests overall in my next commit. I appreciate the suggestion!